packages/asc-ui/src/components/Label/LabelStyle.ts

```diff
@@ -37,10 +42,17 @@ const LabelStyle = styled.label<Props>`
         padding-${position}: 12px;
       }
     `}
+  
+  ${({ active }) =>
+    active &&
+    css`
+      font-weight: 700;
```

Comment:
In the design system there is no reference of setting a font-weight to 700 on a checked checkbox:
![image](https://user-images.githubusercontent.com/7781865/67381302-ffa87480-f58b-11e9-9f23-ad76e738c74c.png)
How do we deal with this?